integrations-and-supported-tools/optuna/scripts/Neptune_Optuna_integration_log_after_study.py

```diff
@@ -1,6 +1,6 @@
 import lightgbm as lgb
-import neptune.new as neptune
-import neptune.new.integrations.optuna as optuna_utils
+import neptune
```

Comment:
Is this intentional? The Optuna notebook seems to import from `neptune.new`

Reply:
Yes. Just pushed changes for optuna. The notebook also imports from Neptune now